src/commands/create/index.ts

```diff
@@ -185,28 +189,12 @@ export default class Create extends Command {
         });
     }
 
-    private async _setDefaultAppManifestInterfaceConfig(interfaces: string[]) {
-        if (this.appManifestInterfaces.interfaces.length > 0) {
+    private async _promptMissingInterfaceConfig(interfaces: string[]): Promise<void> {
```

Comment:
The actual code in this function is not really much. Can it maybe inlined somewhere?